packages/slate-editor/src/lib/scrollIntoView.ts

```diff
@@ -44,11 +44,17 @@ export function scrollIntoView(
         return;
     }
 
-    if (isChildAboveVisibleArea) {
-        const y = parent.scrollTop + elementTop - minTop;
+    const isElementHigherThanParent = elementHeight + minBottom + minTop > parentHeight;
+
+    if (isChildBelowVisibleArea && !isElementHigherThanParent) {
+        const y = parent.scrollTop + elementHeight;
```

Comment:
The math here has a problem.

Previously it was 

```ts
const y = parent.scrollTop + elementTop + elementHeight - parentHeight + minBottom;
```

and now it's just

```ts
const y = parent.scrollTop + elementHeight;
```

This leads to scroll-to following keyboard navigation not displaying blocks properly.

I'm pressing <kbd>↓</kbd> on keyboard multiple times here: 

https://user-images.githubusercontent.com/370680/168607551-30937a8d-c4b4-48cd-8224-dcb569fefd96.mp4

Reply:
Fixed by merging the code from #174